python/pynessie/cli.py

```diff
@@ -101,15 +101,25 @@ def parse_args(self: "DefaultHelp", ctx: click.Context, args: List) -> List:
 @click.option("--json", is_flag=True, help="write output in json format.")
 @click.option("-v", "--verbose", is_flag=True, help="Verbose output.")
 @click.option("--endpoint", help="Optional endpoint, if different from config file.")
+@click.option("--owner", help="Optional repository owner, if different from config file.")
+@click.option("--repo", help="Optional repository name, if different from config file.")
 @click.option("--version", is_flag=True, callback=_print_version, expose_value=False, is_eager=True)
 @click.pass_context
-def cli(ctx: click.core.Context, json: bool, verbose: bool, endpoint: str) -> None:
+def cli(ctx: click.core.Context, json: bool, verbose: bool, endpoint: str, owner: str, repo: str) -> None:
     """Nessie cli tool.
 
     Interact with Nessie branches and tables via the command line
     """
     try:
-        config = build_config({"endpoint": endpoint} if endpoint else None)
+        cfg_map = {}
```

Comment:
They are (or better: can be) in the config file.
I've added some more tests for that case. It turned out, that eagerly creating the `NessieClient` and therefore validating the owner+repo parameters then, doesn't work. For example, if you'd do a `nessie-cli config --add owner snazy`, it would error out saying that the "repo" option is missing. So I'd changed the context-object to build the `NessieClient` lazily.